app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -6,6 +6,7 @@ class ApplicationFormController < CandidateInterfaceController
     def show
       @application_form_presenter = CandidateInterface::ApplicationFormPresenter.new(current_application)
       @application_form = current_application
+      @previous_application_form = current_candidate.application_forms.where.not(id: current_application.id).order(id: :desc).last
```

Comment:
Steve's working on an association between the two applications, probably `.original_application_form`. We can still merge this, and update it later, as long as we make a note somewhere not to forget 👍